tools/webpack.config.dev.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    'webpack-hot-middleware/client?reload=true',
```

Comment:
This is an entry point for development mode.
It will use the stuff from HMR server.
`reload=true` forces the browser to reload the page in case of non-hot-reloadable code